gradle.properties

```diff
@@ -0,0 +1 @@
+org.gradle.parallel=true
```

Comment:
👍 - quick question: it only parallelises the build process and not tests right?

Reply:
parallelises build steps, so each module can be built separately (tests from multiple modules may run concurrently, but within a single module they're run as specified by the test runner)

Reply:
https://guides.gradle.org/performance/#parallel_execution